### ISSUE TITLE
Sort tests in test_libraries alphabetical.

### ISSUE THREAD

**Issue by htgoebel** [pyinstaller/pyinstaller#1636]
This makes it easier to find.

**Comment by giumas** [comment]
If you want, I can do the PR for this (before the PR for adding the `lxml.isoschematron` test: https://github.com/pyinstaller/pyinstaller/issues/1629)

**Comment by htgoebel** [comment]
I changed my mind. Doing this makes it very hard to follow changes.
@giumas Thanks for your offer anyway.

**Comment by giumas** [comment]
@htgoebel OK, so where I should put a new test function? at the beginning or at the end?

**Comment by htgoebel** [comment]
I suggest putting it between keyring and numpy. So at least this test is sorted alphabetical :-)
